services/app-web/src/pages/SubmissionSummary/RateSummary/RateSummary.tsx

```diff
@@ -78,11 +78,14 @@ export const RateSummary = (): React.ReactElement => {
                         <span>&nbsp;Back to dashboard</span>
                     </Link>
                 </div>
-                <SingleRateSummarySection
-                    rate={rate}
-                    isSubmitted // can assume isSubmitted because we are building for CMS users
-                    statePrograms={rate.state.programs}
-                />
+                {loggedInUser?.__typename === 'StateUser' && rate.status === 'UNLOCKED' ?
```

Comment:
Take a peek at `services/app-web/src/pages/SubmissionSideNav/SubmissionSideNav.tsx` line 123 on how we redirect within a page component.